packages/yourloops/lib/patient/patient.hook.tsx

```diff
@@ -89,10 +90,8 @@ export default function usePatientProviderCustomHook(): PatientContextResult {
     return patientTeam.teamId === PRIVATE_TEAM_ID
   }
 
-  const buildPatientFiltersStats = useCallback(() => {
+  const buildPatientFiltersStats = useCallback((): PatientFilterStats => {
     return {
-      all: patientList.filter((patient) => !PatientUtils.isOnlyPendingInvitation(patient)).length,
```

Comment:
I use this is my PR. You could have kept it instead of creating a new concept `pendingPatientsCount`... But nevermind, I will fix this is my PR